aws_jobs/cyhy-data-extract.py

```diff
@@ -375,7 +378,7 @@ def main():
             ES_AWS_CONFIG_SECTION_NAME,
         )
     )
-    json_filename = "{!s}_{!s}.json".format(
+    json_filename = "{}_{!s}.json".format(
```

Comment:
Is there any reason why shouldn't we just do it like this?
  `json_filename = "DMARC_{!s}.json".format(...`

Reply:
None that I can so I have changed it accordingly.